serverV2/contentRouter.js

```diff
@@ -0,0 +1,120 @@
+const express = require('express');
+const path = require('path');
+const utils = require('./utils');
+
+module.exports = () => {
+  const router = express.Router();
+
+  router.get('/:contentType/list', async (req, res) => {
+    const { contentType } = req.params;
+
+    const contentPath = path.resolve('db', `${contentType}.json`);
+    const items = await utils.read(contentPath);
+
+    res.setHeader('Access-Control-Expose-Headers', 'X-Total-Count');
+    res.setHeader('X-Total-Count', items.length);
+    res.send({ data: items, total: items.length });
+  });
+
+  router.get('/:contentType/:itemId/reference', (req, res) => {
+    res.status(501).end();
+  });
+
+  router.get('/:contentType/:itemId', async (req, res) => {
+    const { itemId, contentType } = req.params;
+
+    const contentPath = path.resolve('db', `${contentType}.json`);
```

Comment:
every single content item should be stored in a separate file. otherwise concurrent items editing of the same content type is going to be conflicting. 
So, the db folder structure will look like:
```
/db
  /content-types
    1.json
    2.json
  /articles
    1.json
    2.json
```